tests/test_checkout_integration.py

```diff
@@ -110,3 +112,115 @@ def test_checkout_flow(product_in_stock, client):
     payment = order.payments.latest('pk')
     assert payment.status == 'preauth'
     assert order.status == OrderStatus.NEW
+    latest_address = Address.objects.latest('pk')
+    assert Address.objects.are_identical(order.shipping_address, latest_address)
+
+
+def test_checkout_flow_authenticated_user(product_in_stock, client,
+                                          django_user_model):
```

Comment:
Either use an existing fixture for an authorized client or create one. Please don't set up user accounts inside a test case.